app/models/retrieval.rb

```diff
@@ -115,7 +117,11 @@ def self.process(id, token, env, base_url, access_token)
                 method[:error_message] = []
                 method[:error_message] << error_message
 
-                page_num = results_count / page_size + 1 if Rails.configuration.enable_esi_order_chunking
+                # break the loop
+                if !Rails.configuration.enable_esi_order_chunking || limited_collection?(method['id'])
```

Comment:
```limited_collection...``` part is not necessary since we have already limited the total number of granules to 100 on granule list page. Just to be safe here in case someone uses a bookmarked link to re-order the granules.